### ISSUE TITLE
suggestions for Alarm Clock plugin

### ISSUE THREAD

**Issue by lazka** [quodlibet/quodlibet#707]
Original [issue 707](https://code.google.com/p/quodlibet/issues/detail?id=707) created by murilolima87 on 2011-04-07T20:10:06.000Z:

Hello guys,

I have a suggestion for the Alarm Clock plugin.

I suggest that the user is allowed to specify the following things, besides the alarm time:
(1) volume
(2) a song / album / playlist / etc
(3) order (in order / random / etc, with or without repeat)

That would be specifically useful for me, as I go to sleep and wake up listening to music. When I go to sleep, I listen to slow songs in a low volume, sometimes an album that I like to listen to in order, whereas I like to wake up with loud and happy songs (using the Alarm Clock plugin), and for that I have a playlist which I like to play at random. It is annoying if I have to worry about changing the volume, playlist and playing order before I go to sleep, so I can't just fall asleep listening to music. =P

But I'm not sure if it would be an easy to implement...

Best regards,
Murilo de Lima

**Comment by Beast141** [comment]
Yes i totally agree!!! Right now i can only put the time, yet when i do that, nothing starts up. Does the alarm clock work?  I tried it with the program closed and with the program open and on pause.  it doesnt start any music on given time.

**Comment by Beast141** [comment]
Also, another very important enhancement, in your point number 2) id just like to clarify that there would be an option to choose a favorite radio station you have, and make it start up when the alarm goes off.
